data/saas/config/stripe_config.yml

```diff
@@ -85,8 +85,6 @@ saas_config:
           query_params:
             - name: charge
               value: <charge_id>
-            - name: payment_intent
```

Comment:
Removed `payment_intent` since the way we are building the requests now causes this to be invalid, the Stripe API doesn't let us specify both params at once. The result is still the same since we have a list of disputes that can be identified either by `charge_id` or `payment_intent_id`.